### PR TITLE
Feature v.3

### PR DESCRIPTION
**Pull request by Pozharitskiy**
Refused height's, width's in pixels, redid all block's styles, tried to perform all your guys comments. I'll inspect that code tonight when i will make responsive

### CODE REVIEW

File: scss/components/_folder.scss [hunk 1]
```diff
@@ -1,11 +1,16 @@
 @import "./button";
 @import "./mixins/flex-mix.scss";
 @import "./src/fonts";
+@import "../variables";
 
 %column {
-  width: 372px;
+  width: 31%;
```
Comment (angelinabylitskaya) [inline scss/components/_folder.scss]: Bad value

File: scss/components/_folder.scss [hunk 2]
```diff
@@ -51,13 +58,18 @@
 
 %folder-info {
   padding: 26px 0 10px 0px;
+  font-family: "Open sans";
```
Comment (angelinabylitskaya) [inline scss/components/_folder.scss]: Use variables for fonts

File: scss/style.scss [hunk 3]
```diff
@@ -8,220 +8,236 @@
...
+  justify-content: space-between;
   align-items: flex-end;
 }
 
+.menu-item:hover {
+  border-bottom: 1px solid $second-color;
+  padding-bottom: 4px;
+}
+
+.slider {}
```
Comment (angelinabylitskaya) [inline scss/style.scss]: Do not leave unused code

File: scss/style.scss [hunk 4]
```diff
@@ -8,220 +8,236 @@
...
+
 header li {
```
Comment (angelinabylitskaya) [inline scss/style.scss]: Do not use tags for styling

File: scss/style.scss [hunk 5]
```diff
@@ -8,220 +8,236 @@
...
+.latest-blog-posts {
+  background: $color-primary;
 }
-.paralax-headline {
-  @extend %headline;
+
+.latest-portfolio {}
+
+footer {
+  background: #303030;
```
Comment (angelinabylitskaya) [inline scss/style.scss]: Move all colors to variables

File: scss/style.scss [hunk 6]
```diff
@@ -8,220 +8,236 @@
...
+  color: $color-primary;
+  margin: 7px 0 0 13px;
 }
-.feedback {
-  margin-top: -130px;
-  width: $container-width;
+
+.social-media {
+  left: 0;
+  margin-left: 29%;
```
Comment (angelinabylitskaya) [inline scss/style.scss]: Bad value

**Comment by angelinabylitskaya** [comment]
invalid branch name

File: scss/components/_folder.scss (hunk 1 above)
Comment (Pozharitskiy) [reply scss/components/_folder.scss]: Agree, but unfortunately it's the only way to make this almost PixelPerfect(

File: scss/components/_feedback.scss [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+}
+
+.list-item-text {
+  margin: 0 0 0 65px;
+  line-height: 25px;
+}
+
+.name {
+  width: $container-width * 0.243;
+  font-style: italic;
```
Comment (kobweb) [inline scss/components/_feedback.scss]: Extract common input's styles to `input-box` class, for example `font-style`, `padding `, `height`.
You should follow the DRY principle.

File: scss/style.scss [hunk 8]
```diff
@@ -8,220 +8,236 @@
...
+
+.latest-blog-posts {
+  background: $color-primary;
 }
-.paralax-headline {
-  @extend %headline;
+
+.latest-portfolio {}
+
+footer {
```
Comment (kobweb) [inline scss/style.scss]: Do not use tags!

File: scss/components/_folder.scss (hunk 2 above)
Comment (Pozharitskiy) [reply scss/components/_folder.scss]: DOne

File: scss/style.scss (hunk 3 above)
Comment (Pozharitskiy) [reply scss/style.scss]: Got you

File: scss/style.scss (hunk 4 above)
Comment (Pozharitskiy) [reply scss/style.scss]: I'll fix that

File: scss/components/_feedback.scss (hunk 7 above)
Comment (Pozharitskiy) [reply scss/components/_feedback.scss]: Understood, I will follow that

File: scss/style.scss (hunk 8 above)
Comment (Pozharitskiy) [reply scss/style.scss]: Ok

File: scss/components/_blog.scss [hunk 9]
```diff
@@ -48,31 +63,74 @@
...
+}
+
+.left-blog-arrow {
+  @extend %arrow;
+  width: 28px;
+  height: 28px;
+  border-left: 2px solid $data-color;
+  border-bottom: 2px solid $data-color;
+  transform: rotate(45deg);
+  left: 3%;
```
Comment (kobweb) [inline scss/components/_blog.scss]: You have already had styles `left`, 'transform', `border` at `%arrow` template, haven't you?

File: scss/components/_burger.scss [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+  justify-content: center;
+  align-items: center;
+  line-height: 50px;
+
+  @include screen-small {
+    width: 80vw;
+  }
+}
+
+li {
```
Comment (kobweb) [inline scss/components/_burger.scss]: WTF!?

File: scss/components/_container.scss [hunk 11]
```diff
@@ -0,0 +1,14 @@
+@import "./src/fonts";
+@import "../variables";
+
+a {
```
Comment (kobweb) [inline scss/components/_container.scss]: What it's doing here?

File: scss/components/_footer.scss [hunk 12]
```diff
@@ -0,0 +1,162 @@
+@import "./mixins/flex-mix.scss";
+@import "./src/fonts";
+@import "../variables";
+
+.whole-footer {
+  background: #303030;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  height: 150px;
```
Comment (kobweb) [inline scss/components/_footer.scss]: Don't use fixed height here.

File: scss/components/_latest.scss [hunk 13]
```diff
@@ -0,0 +1,176 @@
...
+.latest-underline {
+  margin-top: 14px;
+}
+
+.latest-underline-headline {
+  @extend %underline-headline;
+  color: $black;
+}
+
+.big-city {
```
Comment (kobweb) [inline scss/components/_latest.scss]: Rename this class

File: scss/components/_burger.scss (hunk 10 above)
Comment (Pozharitskiy) [reply scss/components/_burger.scss]: Renamed

File: scss/components/_container.scss (hunk 11 above)
Comment (Pozharitskiy) [reply scss/components/_container.scss]: my bad

File: scss/components/_latest.scss (hunk 13 above)
Comment (Pozharitskiy) [reply scss/components/_latest.scss]: Done

File: scss/components/_footer.scss (hunk 12 above)
Comment (Pozharitskiy) [reply scss/components/_footer.scss]: Ok

**Pull request merged**